### DISCUSSION TITLE
Show terraform version on command?

### DISCUSSION THREAD

**Discussion by rodhash** [romkatv/powerlevel10k#2789] [General]
Hi

Similarly to what we have for Azure, AWS, GCP, etc.. can we show terraform version only when invoking its command?

Something like this?

`POWERLEVEL9K_ terraform _SHOW_ON_COMMAND`

**Comment by romkatv** [comment]
Yes, you can do that. Show on command is a generic parameter. All segments support it.

```zsh
POWERLEVEL9K_<SEGMENT>[_STATE]_SHOW_ON_COMMAND='<GLOB>'
```

E.g., if you want `terraform` segment to be shown only when you are invoking the `tf` command:

```zsh
POWERLEVEL9K_TERRAFORM_SHOW_ON_COMMAND='tf'
```

**Reply by rodhash** [reply]
Great it worked with: `POWERLEVEL9K_TERRAFORM_VERSION_SHOW_ON_COMMAND`

Terraform itself was already showing up only when invoking its cmd but its version was fixed.

Thank you.
